scripts/particle_rugosity/particle_rugosity.py

```diff
@@ -0,0 +1,68 @@
+# script for calculating the particle rugosity for a given crystal structure (.cif in local working directory, or refcode of a CSD entry)
+# requires a CCDC license, and loading of the CSD-python miniconda environment if run from Linux command line
+
+# import ccdc modules
+import ccdc
+from ccdc.io import CrystalReader
+from ccdc import particle
+from ccdc.particle import Surface
+from ccdc import morphology
+from ccdc.morphology import BFDHMorphology
+
+def particle_rugosity(crystal):
+# load structure, either (1) crystal structure file in working directory, or 
+# (2) CSD refcode
+    head, sep, tail = crystal.partition('.')
+    if sep == '.':
```

Comment:
I'd recommend looking at https://docs.python.org/3/library/os.path.html for handling paths for files, to make it a bit more safe and handle edge cases.

Reply:
Added lines to check if file, try/except, for better input handling